src/Mod/Fem/femsolver/elmer/equations/electrostatic_writer.py

```diff
@@ -125,31 +125,25 @@ def handleElectrostaticBndConditions(self):
                     # output the FreeCAD label as comment
                     if obj.Label:
                         self.write.boundary(name, "! FreeCAD Name", obj.Label)
-                    if obj.PotentialEnabled:
-                        if hasattr(obj, "Potential"):
-                            # Potential was once a float and scaled not fitting SI units
-                            if isinstance(obj.Potential, float):
-                                savePotential = obj.Potential
-                                obj.removeProperty("Potential")
-                                obj.addProperty(
-                                    "App::PropertyElectricPotential",
-                                    "Potential",
-                                    "Parameter",
-                                    "Electric Potential",
-                                )
-                                # scale to match SI units
-                                obj.Potential = savePotential * 1e6
-                            potential = float(obj.Potential.getValueAs("V"))
-                            self.write.boundary(name, "Potential", potential)
```

Comment:
This looks like a migration of some type, is this safe to remove?

Reply:
It is safe. Migration is done in `ConstraintElectrostaticPotential.onDocumentRestored` method.